schemas/Interaction.json

```diff
@@ -45,7 +45,9 @@
         {"const": "WebRoomListRoomsSublistPlusMenuExploreRoomsItem", "description": "User clicked the explore rooms button in the + context menu of the rooms sublist in  Element Web/Desktop." },
 
         {"const": "SpacePanelSelectedSpace", "description": "User tapped the already selected space from the space list."},
-        {"const": "SpacePanelSwitchSpace", "description": "User tapped an unselected space from the space list -> space switching should occur."}
+        {"const": "SpacePanelSwitchSpace", "description": "User tapped an unselected space from the space list -> space switching should occur."},
+        {"const": "RoomLeave", "description": "User tapped on Leave Room button on Room Details screen."},
+        {"const": "RoomAddHome", "description": "User tapped on Add to Home button on Room Details screen."}
```

Comment:
Not sure to understand the description

Reply:
Add home is the button which is only exists on Android I believe. It's adding a shortcut to room to a launcher screen